test/Test/TomlSpec.hs

```diff
@@ -0,0 +1,67 @@
+{-# LANGUAGE RecordWildCards #-}
+
+module Test.TomlSpec where
+
+import Relude (Last (Last), Maybe (Just), Text, pure, ($), (.), (<$>), (<*>))
+
+import Hedgehog (MonadGen, forAll, property, tripping)
+import Test.Tasty (TestTree)
+import Test.Tasty.Hedgehog (testProperty)
+import Toml.Bi.Code (decode, encode)
+
+import Summoner.Config (ConfigP (..), PartialConfig, configT)
+import Summoner.License (License (..), licenseNames)
+import Summoner.ProjectData (CustomPrelude (..), Decision, GhcVer (..))
+
+import qualified Hedgehog.Gen as Gen
+import qualified Hedgehog.Range as Range
+
+genDecision :: MonadGen m => m Decision
+genDecision = Gen.enumBounded
```

Comment:
I think you don't need to write `genDecision` here. You can just import it from `Test.DecisionSpec`.